src/plugins/core/borders.ts

```diff
@@ -151,6 +160,29 @@ export class BordersPlugin extends CorePlugin<BordersPluginState> implements Bor
     return border;
   }
 
+  /**
+   * Return the colors of all borders of a sheet
+   * @param sheetId Id of the sheet that we want to get the borders
+   * @returns All the colors used in the borders of the sheets
+   */
+  getBordersColors(sheetId: UID): Color[] {
```

Comment:
Not directly related to this PR but to custom colors: is it intended to only get colors from the active sheet?

Reply:
It doesn't seem to be the case. Are you sure about it ? I just tried to set a border on a sheet and then switch to another and open the color picker => the custom color from the border is present.

Moreover, in the plugin, we are looping on the sheet id so it should work, shouldn't it ?